repotracker/repotracker_test.go

```diff
@@ -724,14 +724,16 @@ func TestBuildBreakSubscriptions(t *testing.T) {
 	proj2 := model.ProjectRef{
 		Id:                   "proj2",
 		NotifyOnBuildFailure: utility.TruePtr(),
-		Admins:               []string{"u2", "u3"},
+		Admins:               []string{"u2", "u3", "u4"},
 	}
 	u2 := user.DBUser{
 		Id:           "u2",
 		EmailAddress: "shaw@blizzard.com",
 		Settings: user.UserSettings{
+			SlackUsername: "hello.itsme",
+			SlackMemberId: "HELL0173",
```

Comment:
Adding the `SlackMemberId` obfuscates whether the change being tested actually works (the `@` addition) because `SlackMemberId` overwrites `SlackUsername`, so I would remove `SlackMemberId` from this user (you could test `SlackMemberId` with another user if desired).